SkillTrack-Team/skill-track
# Frontend boilerplate

# Description

-Created frontend boilerplate code
-Added React, TypeScript, Webpack, and plugins
-Added frontend file structure and package.json 

## Type of change


# Testing Instructions

After installing npm packages in the client folder, type in terminal npm run build then npm run start. The website should now be visible on http://localhost:9500/.

# Checklist:

- [ ] I have followed the coding style and guidelines of this project.
- [x] I have reviewed my own code to ensure its quality.
- [ ] I have added relevant comments and documentation where necessary.
- [ ] I have updated/added necessary tests for the changes made.
- [ ] I have rebased my branch on top of the latest main (or other target) branch.
